CONTRIBUTING.md

```diff
@@ -1,53 +1,307 @@
-# Contributing to the OpenAPI Specification
+# Contribute to the OpenAPI Specification
 
-***Work in progress!**  Each section links to issues that are relevant to fill out the rest of this document.*
+## Key information
 
-We are currently working on [defining and documenting our new processes](https://github.com/orgs/OAI/projects/5).  Information in this document is up-to-date.  Older _(and sometimes now inaccurate)_ documentation can be found in [DEVELOPMENT.md](DEVELOPMENT.md), which will be removed when everything is updated and documented here.
+This project is covered by our [Code of Conduct](https://github.com/OAI/OpenAPI-Specification?tab=coc-ov-file#readme).
+All participants are expected to read and follow this code.
 
-## Essential Policies
+No changes, however trivial, are ever made to the contents of published specifications (the files in the `versions/` folder).
+Exceptions may be made when links to external URLs have been changed by a 3rd party, in order to keep our documents accurate.
 
-This section serves as a quick guide while we work on the full updated documentation.
+Published versions of the specification are in the `versions/` folder.
+The under-development versions of the specification are in the file `spec/oas.md` on the appropriately-versioned branch.
+For example, work on the next patch release for 3.2 is on `v3.2-dev` in the file `spec/oas.md`.
```

Comment:
Can we use 3.1 as the example here?  The next 3.2 release is 3.2.0, which we would not normally refer to as a "patch release"?  Or just change "patch release" to "release."

Reply:
My logic here was that almost everything needs to go into 3.2 (because we don't update 3.1) and we don't want to update this doc every time so this was the most future-proof option .... I think removing "patch" totally works though!

Reply:
Updated, I _think_ it makes more sense now.